wm323Sup/wm323.proto

```diff
@@ -0,0 +1,52 @@
+
+Terminator = CR;
+
+getType {
+    out "1RS";
+    in "%s %*d %*{CCW|CW} %*{0|1} !";
+    wait 25;
```

Comment:
```# wait 25 is after every command since the manual states thst
# "There should be at least 10mS between commands."
# 25ms was chosen since setting it too close to 10ms might cause problems,
# but it is low enough to be not noticable and slow.```